lib/bambora/client.rb

```diff
@@ -11,10 +12,14 @@ def initialize(options = {})
       yield(self) if block_given?
     end
 
+    def_delegators :connection, :request
+
     # Summary: Payment profiles store confidential payment information. Transactions can be processed against profiles.
     # Docs: https://dev.na.bambora.com/docs/guides/payment_profiles/
     # Endpoint: https://api.na.bambora.com/v1/profiles
-    def profiles; end
+    def profile
+      @profile ||= Bambora::Profile.new(self)
```

Comment:
Passing the Client here so that the same client instance can be used for profiles, payment, etc.